common/lib/xmodule/xmodule/capa_module.py

```diff
@@ -90,6 +95,10 @@ def handle_ajax(self, dispatch, data):
             _, _, traceback_obj = sys.exc_info()  # pylint: disable=redefined-outer-name
             raise ProcessingError, (not_found_error_message, err), traceback_obj
 
+        except TimeExpiredError as err:
+            _, _, traceback_obj = sys.exc_info()  # pylint: disable=redefined-outer-name
+            raise ProcessingError, (time_expired_error_message, err), traceback_obj
```

Comment:
Not sure what recommended is, but I mainly chose this style to remain consistent with the error above it (NotFoundError)